### ISSUE TITLE
Request's source IP is a cell's IP instead of a VXLAN one.

### ISSUE THREAD

**Issue by b10s** [cloudfoundry/cf-networking-release#37]
## Issue

In case of two app's  located on the same cell source IP will be a cell's IP.

## Context

During the cf-networking test we realaized that sometime source IP's of requests have different from VXLAN subnet. We find out that connection between application instances located on the same cell will use cell IP, not contaner VXLAN IP. At the same time connection between application instances located on different cells will be with VXLAN source IP.

## Steps to Reproduce
```
$ cf network-policies
Listing network policies in org my-org / space my-space as user@example.com...

source     destination   protocol   ports
feeder     sd            tcp        1234

$ cf-instances.sh sd
Application: sd (1495e2c6-91dd-45f1-bc94-df608ae4d83f)
Index  State    Uptime  IP             Port   CPU  Mem    Mem_Quota  Disk   Disk_Quota
0      RUNNING  9531s   100.73.61.131  61001  2%   23.8M  64M        94.7M  1024M

$ cf-instances.sh feeder
Application: feeder (7bd85784-b7b4-4673-84d3-1b43c8fc0ba9)
Index  State    Uptime  IP             Port   CPU   Mem    Mem_Quota  Disk    Disk_Quota
0      RUNNING  6392s   100.73.61.131  61007  0.3%  27.5M  64M        105.9M  1024M
1      RUNNING  7798s   100.73.61.130  61001  0.2%  28.1M  64M        105.9M  1024M
2      RUNNING  7798s   100.73.61.129  61038  0%    23.3M  64M        105.9M  1024M



########## CONSOLE_1 ##########
$ cf ssh sd

# TIME 1.1
vcap@bdb549a1-2399-4847-5a3b-73bf:~$ nc -vl 1234
Listening on [0.0.0.0] (family 0, port 1234)
Connection from [100.73.61.131] port 1234 [tcp/*] accepted (family 2, sport 33758)

# TIME 2.1
vcap@bdb549a1-2399-4847-5a3b-73bf:~$ nc -vl 1234
Listening on [0.0.0.0] (family 0, port 1234)
Connection from [10.78.184.42] port 1234 [tcp/*] accepted (family 2, sport 38524)
###############################



########## CONSOLE_2 ##########
$ cf ssh feeder -i 0

# TIME 1.2
vcap@46c732fe-428a-4c68-6a42-7c47:~$ nc -zv 10.79.215.6 1234
Connection to 10.79.215.6 1234 port [tcp/*] succeeded!

vcap@46c732fe-428a-4c68-6a42-7c47:~$ exit

# TIME 2.2
$ cf ssh feeder -i 1
vcap@3fb455bf-1233-4256-576a-6142:~$ nc -zv 10.79.215.6 1234
Connection to 10.79.215.6 1234 port [tcp/*] succeeded!

vcap@3fb455bf-1233-4256-576a-6142:~$ exit
###############################
```
## Expected result

We expect to have VXLAN IP as a source IP in all cases.

**Comment by christianang** [comment]
We took a look and fixed the issue in https://github.com/cloudfoundry/silk-release/commit/d2f9b64b20ef0949a5aff2d03944cff6f735186c. This will be in the next release of silk-release.

Thanks,
Christian and Aidan

**Comment by rosenhouse** [comment]
Fixed in [Silk Release v2.0.0](https://github.com/cloudfoundry/silk-release/releases/tag/v2.0.0) + [CF Networking Release v2.0.0](https://github.com/cloudfoundry/cf-networking-release/releases/tag/v2.0.0)

**Comment by tylerschultz** [comment]
@b10s, 

Hello from the networking team. We are working on a feature set that would seemingly be interesting to you. We are wondering if you're willing to share your use case for this bug? Why is it important that all traffic come from the overlay? 

We ask because we are working on features that may affect the source IP. 

Thanks,

@tylerschultz
